app/controllers/api/v1/books_controller.rb

```diff
@@ -0,0 +1,15 @@
+module Api
+  module V1
+    class BooksController < ApplicationController
+      def index
+        @books = Book.all
+        render_paginated json: @books
+      end
+
+      def show
+        @book = Book.find(params[:id])
```

Comment:
this can be raise an exception if book searched is not found, please catch the exception with `rescue_from` on application_controller. https://apidock.com/rails/ActiveSupport/Rescuable/ClassMethods/rescue_from

Reply:
Done